apps/desktop/src/renderer/globals.css

```
@@ -6,6 +6,17 @@
 @source "../../../../packages/ui/src/**/*.{ts,tsx}";
 @source "../../../../packages/ui/node_modules/streamdown/dist/*.js";
 
+/*
+ * Active when the sidebar workspace activity strip should show its expanded
+ * (pill) state: the workspace item (`group/item`) is hovered or keyboard
+ * focused, or one of the strip's own dropdown menus (`group/details` scope)
+ * is open so it can't collapse under the pointer.
+ */
+@custom-variant details-expanded (&:is(
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)